mappings/net/minecraft/world/chunk/UpgradeData.mapping

```diff
@@ -1,16 +1,31 @@
 CLASS net/minecraft/class_2843 net/minecraft/world/chunk/UpgradeData
 	FIELD field_12950 NO_UPGRADE_DATA Lnet/minecraft/class_2843;
-	FIELD field_12951 sides Ljava/util/EnumSet;
+	FIELD field_12951 sidesToUpgrade Ljava/util/EnumSet;
 	FIELD field_12952 EIGHT_WAYS [Lnet/minecraft/class_2355;
 	FIELD field_12953 BLOCK_TO_LOGIC Ljava/util/Map;
 	FIELD field_12954 CALLBACK_LOGICS Ljava/util/Set;
-	FIELD field_12955 indices [[I
+	FIELD field_12955 centerIndicesToUpgrade [[I
 	FIELD field_12956 LOGGER Lorg/apache/logging/log4j/Logger;
 	METHOD <init> (Lnet/minecraft/class_2487;)V
 		ARG 1 tag
+	METHOD method_12348 upgradeCenter (Lnet/minecraft/class_2818;)V
+		ARG 1 chunk
+	METHOD method_12349 isDone ()Z
 	METHOD method_12350 toTag ()Lnet/minecraft/class_2487;
+	METHOD method_12351 applyAdjacentBlock (Lnet/minecraft/class_2680;Lnet/minecraft/class_2350;Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;Lnet/minecraft/class_2338;)Lnet/minecraft/class_2680;
+		ARG 0 oldState
+		ARG 1 dir
+		ARG 2 world
+		ARG 3 currentPos
+		ARG 4 otherPos
+	METHOD method_12352 upgradeSide (Lnet/minecraft/class_2818;Lnet/minecraft/class_2355;)V
+		ARG 0 chunk
+		ARG 1 side
+	METHOD method_12356 postProcessChunk (Lnet/minecraft/class_2818;)V
```

Comment:
```suggestion
	METHOD method_12356 postProcess (Lnet/minecraft/class_2818;)V
```

Reply:
Honestly I don't really like the name post process, it doesn't really describe the purpose of the method, only when it's called. Perhaps `fixChunk` (or just `fix`) might be a better name?

Reply:
I renamed it to `upgrade`